src/Symfony/Bundle/DependencyInjection/ApiPlatformExtension.php

```diff
@@ -103,13 +103,20 @@ public function load(array $configs, ContainerBuilder $container): void
         $formats = $this->getFormats($config['formats']);
         $patchFormats = $this->getFormats($config['patch_formats']);
         $errorFormats = $this->getFormats($config['error_formats']);
+        $docsFormats = $this->getFormats($config['docs_formats']);
 
         // Backward Compatibility layer
         if (isset($formats['jsonapi']) && !isset($patchFormats['jsonapi'])) {
             $patchFormats['jsonapi'] = ['application/vnd.api+json'];
         }
 
-        $this->registerCommonConfiguration($container, $config, $loader, $formats, $patchFormats, $errorFormats);
+        if (!isset($docsFormats['jsonopenapi'], $docsFormats['json'], $docsFormats['yamlopenapi'])) {
```

Comment:
Shouldn't we use the default values in the configuration class instead of forcing default values here? What if the user want to disable these formats?

Reply:
Good question, what do you think @soyuka ?

Reply:
this is only a bc layer when only `json` is set we need to set `jsonopenapi` to make it work no ?